tests/CMakeLists.txt

```
@@ -6,6 +6,9 @@ add_executable(${PROJECT_NAME} directory.cpp file.cpp filesystem.cpp)
 target_link_libraries(${PROJECT_NAME} tmp::tmp GTest::gtest_main)
 target_compile_definitions(
     ${PROJECT_NAME} PRIVATE PREFIX="com.github.bugdea1er.tmp")
+if(TMP_MODULES)
+    target_compile_definitions(${PROJECT_NAME} PRIVATE TMP_USE_MODULES)
+endif()
```

```suggestion
target_compile_definitions(${PROJECT_NAME} PRIVATE "$<$<BOOL:${TMP_MODULES}>:TMP_USE_MODULES>")
```
And maybe it would be better to use the same variable name and macro name

---

Done both